pkg/printers/internalversion/printers.go

```diff
@@ -952,9 +951,7 @@ func printPod(pod *api.Pod, options printers.GenerateOptions) ([]metav1.TableRow
 		}
 	}
 
-	if pod.DeletionTimestamp != nil && pod.Status.Reason == node.NodeUnreachablePodReason {
-		reason = "Unknown"
-	} else if pod.DeletionTimestamp != nil {
```

Comment:
@chenchun Can you explain how this change is related to this PR?

Reply:
These lines are introduced by https://github.com/kubernetes/kubernetes/pull/36017.
`node.NodeUnreachablePodReason` is only assigned to `pod.Status.Reason` in the deleted function `DeletePods` from pkg/controller/util/node/controller_utils.go of this PR, so the kubectl part can be cleaned either.

Reply:
Makes sense to me now, thank you.